### PR TITLE
Sosivio DataDog Integration

### PR DESCRIPTION
**Pull request by DanArlowski**
### What does this PR do?

Sosivio Integration To Datadog

### Motivation

Allowing DataDog Customers to see Sosivio data in their Datadog Panel

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: sosivio/README.md
Comment (buraizu) [inline sosivio/README.md:5]:
```suggestion
Integrate Sosivio with Datadog to send your Sosivio alerts and notifications directly to Datadog.
```

File: sosivio/README.md
Comment (buraizu) [inline sosivio/README.md:12]:
```suggestion
2. Click **Install**.

For more information, view the [Sosivio documentation][3].
```

File: sosivio/README.md
Comment (buraizu) [inline sosivio/README.md:19]:
```suggestion
Need help? Contact [Datadog support][1] or reach out to [Sosivio](mailto:info@sosiv.io).
```

File: sosivio/README.md
Comment (buraizu) [inline sosivio/README.md:23]:
```suggestion
[2]: https://docs.datadoghq.com/account_management/api-app-keys/#add-an-api-key-or-client-token
[3]: https://docs.sosiv.io
[4]: https://docs.datadoghq.com/getting_started/site/
```

File: sosivio/README.md
Comment (buraizu) [inline sosivio/README.md:11]:
```suggestion
1. In the Sosivio Dashboard Configuration page, add the [Datadog API key][2] and the Datadog URL (if not the default datadoghq.com site). See [Datadog sites][4] to learn more.
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thank you for the PR! Made some edits to better align the page with our contributing guidelines:
https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md

Feel free to reach out with any questions!

File: sosivio/README.md
Comment (anawishnoff) [inline sosivio/README.md:5]: It may be helpful to add a bit more detail about what Sosivio does, and the types of data being sent into Datadog

File: sosivio/images/IMAGES_README.md
Comment (anawishnoff) [inline sosivio/images/IMAGES_README.md:1]: This file is meant as a guideline for your own reference and can be deleted

File: sosivio/manifest.json
Comment (anawishnoff) [inline sosivio/manifest.json:14]: A description is required

File: sosivio/manifest.json
Comment (sarah-witt) [inline sosivio/manifest.json:13]: Can you add a `CHANGELOG.md` file?

**Comment by DanArlowski** [comment]
hey guys! thanks for taking the time to review that

unfortunately, im a bit overwhelmed this week but promise to follow up on this next week and align the issues


anything that is related to "public" (i.e - photos, etc) 
we want to deploy that into the environment provded by datadog first in order to snapshot the correct images :)


Thanks!

File: sosivio/manifest.json
Comment (DanArlowski) [reply sosivio/manifest.json:16]: We'll do images once we have the event type "sosivio" :+1:  we want to images to be as accurate as possible.
thats why its still not public :)

**Comment by DanArlowski** [comment]
waiting internally to get a shorter than 70 chars description :shrug:

**Comment by DanArlowski** [comment]
thanks for your time,
@anawishnoff @sarah-witt @buraizu 


Im facing issues with the validation,
`manifest.json` is valid json, but no specific error is getting printed (besides the usual `no dd_url`




```
(venv) ➜  integrations-extras git:(Sosivio_integration) ddev validate  all
`integrations-extras` directory `integrations-extras` does not exist, defaulting to the current location.
Running validations for extras repo ...
---
Skipping agent-reqs
---
Executing validation ci
Validating CI Configuration...
Success
Validating Code Coverage Configuration...
Success

---
Executing validation codeowners
All integrations have valid codeowners.

---
Executing validation config
Validating default configuration files for 52 checks...
All 85 configuration files are valid!

---
Executing validation dashboards
Validating Dashboard definition files for 124 checks...
58 valid files

---
Skipping dep
---
Skipping eula
---
Executing validation jmx-metrics
Validating JMX metrics files for 4 checks ...
4 total JMX integrations
4 valid metrics files

---
Skipping http
---
Executing validation imports
Validating imports for 124 checks to avoid deprecated modules ...
Validation passed!

---
...

...
sosivio/manifest.json... FAILED
  skipping check for changed fields: integration not on default branch
  sosivio: `display_on_public_website` is disabled, set to `True` if you want the integration documentation to be published.
No `dd_url` has been set for org `default`
...
2 valid files
5 checks with warnings
16 invalid filessqreen/manifest.json... WARNING
...

(venv) ➜  integrations-extras git:(Sosivio_integration) 
```
(`...` represent  logs unrelated to this PR)

can you guys help me spot what am I missing?

**Comment by steveny91** [comment]
@DanArlowski Can you check if you have this in you ddev config:

```
[orgs.default]
dd_url = "https://app.datadoghq.com"
```

The command is `ddev config edit`. Seems like one of the validation is trying to reach our API to get a response to validate that part of the manifest

**Comment by DanArlowski** [comment]
@steveny91  - as you can see - it fails on the Github validations, too

as you can see - no files have been touched in the PR except for `CODEOWNERS` and the sosivio integration folder.

That is why im raising it here, too :) as the logs are not clear regarding the problem

the dd_url was raised every time I ran the validation, and IIRC -  its labeled as warning if not on the Default branch of the DataDog repo(this one), so i don't think thats the case

File: sosivio/README.md
Comment (buraizu) [inline sosivio/README.md:13]:
```suggestion
```

**Review by buraizu — CHANGES_REQUESTED** [review]

**Comment by DanArlowski** [comment]
@steveny91 @anawishnoff @yzhan289 @sarah-witt  - still waiting for help on why it fails :) 



seems like the cause is the classifier tags:
```json
  "classifier_tags": [
    "Supported OS::Linux",
    "Category::Notification",
    "Category::Monitoring",
    "Category::Containers",
    "Category::Network",
    "Category::Orchestration",
    "Category::Alerting"
    ],
```
which seems correct to me (even when keeping on the supporte os - it still fails.


BTW - seems like im not the only project failing on these (locally)
```
Validating manifest.json files for 124 checks ...
apptrail/manifest.json... FAILED
Error validating manifest schema:
API input validation failed: {'tile': {'classifier_tags': [u'Missing data for required field.']}}
data_runner/manifest.json... FAILED
Error validating manifest schema:
API input validation failed: {'tile': {'classifier_tags': [u'Missing data for required field.']}}
embrace_mobile/manifest.json... FAILED
Error validating manifest schema:
API input validation failed: {'tile': {'classifier_tags': [u'Missing data for required field.']}, 'author': {'sales_email': [u'Missing data for required field.']}}
eversql/manifest.json... FAILED
Error validating manifest schema:
API input validation failed: {'tile': {'classifier_tags': [u'Missing data for required field.']}}
fairwinds_insights/manifest.json... FAILED
Error validating manifest schema:
API input validation failed: {'tile': {'classifier_tags': [u'Missing data for required field.'], 'changelog': [u'Missing data for required field.']}}
finout/manifest.json... FAILED
  finout: `display_on_public_website` is disabled, set to `True` if you want the integration documentation to be published.
Error validating manifest schema:
API input validation failed: {'tile': {'classifier_tags': [u'Missing data for required field.']}}
Skipping validation 


 since errors have already been found.
harness_cloud_cost_management/manifest.json... FAILED
  The tile description is 124 characters long. It should be no longer than 70 characters.
launchdarkly/manifest.json... FAILED
Error validating manifest schema:
API input validation failed: {'tile': {'classifier_tags': [u'Missing data for required field.']}}
lightstep_incident_response/manifest.json... FAILED
Error validating manifest schema:
API input validation failed: {'tile': {'classifier_tags': [u'Missing data for required field.']}, 'assets': {'integration': {'metrics': {'check': [u'Invalid type.']}}}}
pagerduty/manifest.json... FAILED
Error validating manifest schema:
API input validation failed: {'tile': {'classifier_tags': [u'Missing data for required field.']}}
planetscale/manifest.json... FAILED
  planetscale: `display_on_public_website` is disabled, set to `True` if you want the integration documentation to be published.
Error validating manifest schema:
API input validation failed: {'tile': {'classifier_tags': [u'Missing data for required field.']}}
purefa/manifest.json... FAILED
Error validating manifest schema:
API input validation failed: {'tile': {'classifier_tags': [u'Missing data for required field.']}, 'author': {'sales_email': [u'Missing data for required field.']}}
rookout/manifest.json... FAILED
Error validating manifest schema:
API input validation failed: {'tile': {'classifier_tags': [u'Missing data for required field.'], 'changelog': [u'Missing data for required field.']}, 'assets': {'integration': {'metrics': {'check': [u'Invalid type.']}}}, 'author': {'sales_email': [u'Missing data for required field.']}}
sedai/manifest.json... WARNING
  sedai: `is_public` is disabled, set to `True` if you want the integration documentation to be published.
shoreline/manifest.json... FAILED
Error validating manifest schema:
API input validation failed: {'tile': {'classifier_tags': [u'Missing data for required field.']}}
sosivio/manifest.json... FAILED
  skipping check for changed fields: integration not on default branch
  sosivio: `display_on_public_website` is disabled, set to `True` if you want the integration documentation to be published.
Error validating manifest schema:
API input validation failed: {'tile': {'classifier_tags': [u'Missing data for required field.']}}
sqreen/manifest.json... WARNING
  sqreen: `is_public` is disabled, set to `True` if you want the integration documentation to be published.
twenty_forty_eight/manifest.json... FAILED
Error validating manifest schema:
API input validation failed: {'tile': {'classifier_tags': [u'Missing data for required field.']}}
``` 


thanks!

File: sosivio/README.md
Comment (buraizu) [inline sosivio/README.md:11]:
```suggestion
If you don’t have a Sosivio account set up, [create an account][5] and sign up for a free 4-week trial of Sosivio Premium directly from our website (no credit card required). At the end of the 4-week trial, your license converts to the Sosivio Community Version, which is free, forever. With the account set up, you are ready to start using the Sosivio Application in Datadog. 
```

File: sosivio/README.md
Comment (buraizu) [inline sosivio/README.md:14]:
```suggestion
Sosivio is installed under one namespace (labeled "sosivio") which creates all the required components for the product. 
```

**Review by buraizu — APPROVED** [review]
Thanks, made a couple of additional minor suggestions, but LGTM

File: sosivio/README.md
Comment (DanArlowski) [reply sosivio/README.md]: Thanks for the thorough reading :)

File: sosivio/manifest.json
Comment (DanArlowski) [reply sosivio/manifest.json]: thanks!
a more in-depth log would've seem good here :+1:

File: sosivio/README.md
Comment (DanArlowski) [reply sosivio/README.md]: FYI - that caused a failure:
```
Run ddev validate readmes $CHANGED
Error:     readme contains non-ASCII character(s)
    | Sosivio’s Datadog integration allows users to see Sosivio Failure Alerts directly in Datadog dashboards and instantly be redirected to the Sosivio UI to remediate those failures (with Sosivio Premium license). You can also add context around Datadog’s signals with Sosivio’s root cause determination for critical failures. 
    |        ^                                                                                                                                                                                                                                                 ^                      ^           
``` 

probably due to the line breaks on your Mac/Windows machine.
fixed that - just letting you know :)

**Comment by DanArlowski** [comment]
@fanny-jiang  @yzhan289  - waiting for your approval :)

File: sosivio/README.md
Comment (anawishnoff) [inline sosivio/README.md:11]:
```suggestion
If you don't have a Sosivio account, [create an account][5] and sign up for a free 4-week trial of Sosivio Premium directly from our website (no credit card required). At the end of the 4-week trial, your license converts to the Sosivio Community Version, which is free, forever. With the account set up, you are ready to start using the Sosivio integration in Datadog.
```

File: sosivio/README.md
Comment (anawishnoff) [inline sosivio/README.md:19]:
```suggestion
1. In the Sosivio Dashboard Configuration page, add your [Datadog API key][2] and Datadog URL (if not the default datadoghq.com site). See [Datadog sites][4] to learn more.
```

Comment (anawishnoff) [inline sosivio/README.md:19]: Maybe a hyperlink to the dashboard configuration page would be helpful here?

**Review by fanny-jiang — APPROVED** [review]

File: sosivio/README.md
Comment (DanArlowski) [reply sosivio/README.md:7]: Datadog Agents mostly

File: sosivio/README.md
Comment (DanArlowski) [reply sosivio/README.md]: Dashboard is dynamic by deployment (on-prem)

**Comment by DanArlowski** [comment]
waiting for you approval to continue from our side :)

**Review by yzhan289 — APPROVED** [review]

File: sosivio/manifest.json
Comment (anawishnoff) [inline sosivio/manifest.json:13]: @DanArlowski please add at least one image to your tile - for example, it can be a screenshot of the Datadog dashboard or the Sosivio UI. To add an image, use the following syntax:

```
"media_type": "image",
"caption": "caption goes here",
"image_url": "relative-path-to-image.png"
```

**Review by anawishnoff — CHANGES_REQUESTED** [review]
Looks good other than the pending images!

Internally, we're also still waiting on getting these logos uploaded (nothing needed on your end here, and this shouldn't take too long)

File: sosivio/manifest.json
Comment (DanArlowski) [reply sosivio/manifest.json:13]: @anawishnoff , I've added one image - regardless - once we'll do a public `true` we'll add more :)

Comment (DanArlowski) [reply sosivio/manifest.json]: @anawishnoff  - not sure how i missed the image,
added it now

and im aware.
thanks!

**Review by anawishnoff — APPROVED** [review]

**Pull request merged**
